lib/WorseReflection/Tests/SelfTest/assert/assert-negate-instanceof1.test

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class F {
+    /** @param Foo|Bar $foo */
+    public function foo($foo) {
```

Comment:
note: create a ticket as defining a simple `function` doesn't inherit the `@param` docblocks